### PR TITLE
python3-mysqlclient: update to 2.1.0.

### PR DESCRIPTION
**Pull request by hamlolum**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/python3-mysqlclient/template [hunk 1]
```diff
@@ -10,6 +10,6 @@ depends="python3"
 short_desc="Python3 interface to MySQL/MariaDB"
 maintainer="Alin Dobre <alin.dobre@outlook.com>"
 license="GPL-2.0-or-later"
-homepage="https://github.com/PyMySQL/mysqlclient-python"
-distfiles="${PYPI_SITE}/m/mysqlclient/mysqlclient-${version}.tar.gz"
-checksum=3981ae9ce545901a36a8b7aed76ed02960a429f75dc53b7ad77fb2f9ab7cd56b
+homepage="https://github.com/PyMySQL/mysqlclient"
+distfiles="https://github.com/PyMySQL/mysqlclient/archive/refs/tags/v${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/python3-mysqlclient/template:14]:
```suggestion
homepage="https://github.com/PyMySQL/mysqlclient"
changelog="https://raw.githubusercontent.com/PyMySQL/mysqlclient/main/HISTORY.rst"
distfiles="https://github.com/PyMySQL/mysqlclient/archive/refs/tags/v${version}.tar.gz"
```
why did you decide to switch to a github tarball?

**Review by paper42 — COMMENTED** [review]
Could you describe how you tested this PR?

File: srcpkgs/python3-mysqlclient/template (hunk 1 above)
Comment (hamlolum) [reply srcpkgs/python3-mysqlclient/template:14]: It seemed neater.

**Comment by hamlolum** [comment]
I have been using v2.1.0 for the past couple of weeks and it works fine.

File: srcpkgs/python3-mysqlclient/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/python3-mysqlclient/template]: If it isn't objectively better (for example containing tests), I think we should just use the pypi tarball.

**Comment by paper42** [comment]
> I have been using v2.1.0 for the past couple of weeks and it works fine.

This is a library used by soci in the repositories, so we should test this. But I noticed the source code doesn't even mention python3-mysqlclient, is this a mistake? @Johnnynator

**Comment by Johnnynator** [comment]
> > I have been using v2.1.0 for the past couple of weeks and it works fine.
> 
> This is a library used by soci in the repositories, so we should test this. But I noticed the source code doesn't even mention python3-mysqlclient, is this a mistake? @Johnnynator

Can be ignored, since soci tests can't be run without a complicated setup, it was needed during checks for some script or by some script provided by mariadb. I can't remember the exact reason right now.

File: srcpkgs/python3-mysqlclient/template (hunk 1 above)
Comment (hamlolum) [reply srcpkgs/python3-mysqlclient/template]: I have changed it back to the pypi tarball.

**Pull request closed without merging**